izanami-frontend/src/pages/users.tsx

```diff
@@ -82,10 +83,25 @@ export function Users() {
       },
     }
   );
-
   const inviteUserMutation = useMutation(
-    (data: { email: string; admin: boolean; rights: TRights }) =>
-      createInvitation(data.email, data.admin, data.rights)
+    (data: {
+      admin: boolean;
+      email: string;
+      rights: TRights;
+      userToCopy: string;
+    }) => {
+      if (data.userToCopy) {
+        return queryUser(data.userToCopy).then((res: TUser) => {
+          const updatedData = { ...data, rights: res.rights };
```

Comment:
This doesn't work when copying admin user, since admin information is at the user root, not in rights subobject.